config/google_cloud_exporter/iis/config.yaml

```diff
@@ -0,0 +1,30 @@
+receivers:
+  iis:
+    collection_interval: 10s
+
+processors:
+
+  normalizesums:
+
+  batch:
+
+exporters:
+  googlecloud:
+    retry_on_failure:
+      enabled: false
+    metric:
+      prefix: workload.googleapis.com
+    resource_mappings:
+      - source_type: ""
+        target_type: generic_node
```

Comment:
Now that v0.6.0 is released, we can simplify these a LOT. The exporter now has:
- Built in normalize sums processor (doesn't need to be configured separately)
- Built in Batch processor (doesn't need to be configured separately)
- Automatic mapping to generic_node
- metric prefix is default set to "workload.googleapis.com"
- retry_on_failure is default disabled

I think from now on, we should try using the new exporter defaults with these configs.

Let me know if any of that was unclear.

https://github.com/observIQ/observiq-otel-collector/tree/main/exporter/googlecloudexporter